htdocs/compta/facture/class/api_invoices.class.php

```diff
@@ -421,7 +442,7 @@ function delete($id)
      * @param int   $id             Id of invoice
      * @param array $request_data   InvoiceLine data
      *
-     * @url     POST {id}/lines
+     * @url     POST {id}/addline
```

Comment:
We must keep the {id}/lines.
The action is ported by the "POST" (POST = add, PUT = update)